packages/@sanity/cli/package.json

```diff
@@ -62,6 +62,8 @@
   },
   "dependencies": {
     "@babel/traverse": "^7.19.0",
+    "@vercel/frameworks": "^1.3.2",
+    "@vercel/fs-detectors": "^3.8.4",
```

Comment:
We need to clarify the license and stability of these packages with Vercel before we can merge this. They are shipped with Apache 2 license files, but are marked as `UNLICENSED` and `MIT` in the `package.json` files. I've opened a PR to address the UNLICENSED case, but not sure what the correct value is for the other: https://github.com/vercel/vercel/pull/9696

As for the stability: we should either ensure that these are semver stable, or pin them.